install/streams.csv

```diff
@@ -1316,8 +1330,8 @@ MWFS,20,B,N,50,true,false,false,2014-11-03T00:01:00Z,2020-06-16T00:00:00Z
 MWFS,20,H,N,200,true,false,true,2006-07-21T01:00:00Z,9999-01-01T00:00:00Z
 MWZ,10,L,H,1,false,false,false,2004-02-29T00:00:00Z,9999-01-01T00:00:00Z
 MWZ,10,H,H,100,false,false,false,2004-02-29T00:00:00Z,9999-01-01T00:00:00Z
-MWZ,20,B,N,50,false,false,false,2004-02-29T00:00:00Z,2011-08-18T20:50:00Z
-MWZ,20,H,N,200,false,false,true,2004-02-29T00:00:00Z,2011-08-18T20:50:00Z
+MWZ,20,B,N,50,false,false,false,2004-02-29T00:00:00Z,2011-08-18T12:00:00Z
+MWZ,20,H,N,200,false,false,true,2004-02-29T00:00:00Z,2011-08-18T12:00:00Z
```

Comment:
shall the end timestamp for MWZ be also changed in `sensors.csv` ?
currently this is
`Kinemetrics Inc.,FBA-ES-T-ISO,3498,MWZ,20,0,,0,0,0,0,0,0,2011-03-23T02:00:00Z,2011-08-18T20:50:00Z`

Reply:
Yes, possibly. The sensor time could be taken from when it was physically removed whereas the stream could be from when it stopped recording data. It sort of depends on a policy with which way to go (a sensor that doesn't record anything doesn't need to know when it was removed, that is we're not tracking assets anymore).